### PR TITLE
Fix incorrect ms.service metadata values

### PR DESCRIPTION
**Pull request by ntrogh**

### CODE REVIEW

**Comment by ntrogh** [comment]
#sign-off

**Comment by ntrogh** [comment]
@danieljurek can you review these metadata changes and related fixes?

File: docs-ref-services/preview/mgmt-devtestlabs-readme.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+title: DevTest Labs Azure SDK for Python
+description: Learn how to manage Azure DevTest Labs resources with the Azure SDK for Python.
 keywords: Azure, python, SDK, API, azure-mgmt-devtestlabs, devtestlabs
 author: lmazuel
 ms.author: lmazuel
 ms.date: 10/31/2022
 ms.topic: reference
 ms.devlang: python
-ms.service: devtestlabs
+ms.service: devtest-lab
```
Comment (danieljurek) [inline docs-ref-services/preview/mgmt-devtestlabs-readme.md:10]: This is a package-level overview (filename ends in -readme.md) and its metadata is generated by automation. This metadata will be overwritten on the next release of this package. To make this change persist you also need to open a PR to the metadata CSV file here: https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv

Set the value of MSDocService -- https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/602/Package-Onboarding-step-by-step?anchor=docs-metadata

Comment (ntrogh) [reply docs-ref-services/preview/mgmt-devtestlabs-readme.md:10]: @danieljurek Should I update the corresponding packages file for Python in this case?

Comment (danieljurek) [reply docs-ref-services/preview/mgmt-devtestlabs-readme.md:10]: Yes, the CSV link should've been: https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/python-packages.csv

Comment (ntrogh) [reply docs-ref-services/preview/mgmt-devtestlabs-readme.md:10]: Tagged you in the PR. Thanks a lot for your support!

**Review by danieljurek — APPROVED** [review]

**Pull request merged**